src/Microsoft.FeatureManagement/IFeatureManagementBuilder.cs

```diff
@@ -30,5 +31,12 @@ public interface IFeatureManagementBuilder
         /// <typeparam name="T">An implementation of <see cref="ISessionManager"/></typeparam>
         /// <returns>The feature management builder.</returns>
         IFeatureManagementBuilder AddSessionManager<T>() where T : ISessionManager;
+
+        /// <summary>
+        /// Adds an <see cref="ITargetingContextAccessor"/> to be used for targeting.
+        /// </summary>
+        /// <typeparam name="T">An implementation of <see cref="ITargetingContextAccessor"/></typeparam>
+        /// <returns>The feature management builder.</returns>
+        IFeatureManagementBuilder WithTargeting<T>() where T : ITargetingContextAccessor;
```

Comment:
Adding a method on an interface is a breaking change. This should be an extension method. I believe we already have `IFeatureManagementBuilderExtensions`